### ISSUE TITLE
Multiple CLI Reporters

### ISSUE THREAD

**Issue by helloanoop** [usebruno/bruno#2918]
Add support for multiple CLI Reporters.

As a user, I want my collection to produce multiple reports (json, HTML, JUnit)

### Current API
```bash
# json report
bru run request.bru --output results.json

# junit report 
bru run request.bru --output results.xml --format junit

#  html report
bru run request.bru --output results.html --format html 
```

### Proposed API
```bash
# json report
bru run request.bru --reporter-json results.json

# json and junit report
bru run request.bru --reporter-json results.json  --reporter-junit results.xml
```

Past Issues / Discussions
* #2045
* https://github.com/usebruno/bruno/discussions/1068#discussioncomment-8718094

Related PRs
* #2911

**Comment by iamdavidfrancis** [comment]
I think this it much cleaner than my implementation. I can push an update to my PR with this as the schema.

**Comment by helloanoop** [comment]
This is now available `v1.29.0` onwards.
